CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ project(mender)
 # set(CMAKE_VERBOSE_MAKEFILE ON)
 enable_testing()
 
+option(COVERAGE "Turn coverage instrumentation on (Default: OFF)" OFF)
+if($CACHE{COVERAGE})
+  set(CMAKE_CXX_FLAGS "--coverage $CACHE{CMAKE_CXX_FLAGS}")
```

Comment:
Isn't `--coverage` a linker flag as well?

Reply:
I don't think so. It's not mentioned on the `ld` man page.

Reply:
Hmm, indeed. Looking at the output from the Makefile, it looks like CMake passes the `--coverage` argument to the linker even if it is only in `CMAKE_CXX_FLAGS`. And it's accepted. That's two unexpected behaviors for the price of one, but the net result is that we don't seem to need any more flags.

Reply:
Great! And "Great..."!